.github/workflows/ci.yaml

```diff
@@ -105,7 +105,7 @@ jobs:
           NETWORK: ${{ matrix.NETWORK }}
           PYTEST_OPTS: ${{ matrix.PYTEST_OPTS }}
           PYTEST_PAR: ${{ matrix.PYTEST_PAR }}
-          TEST_CMD: ${{ matrix.TEST_CMD }}
+          TEST_CMD: "make check-protos"
```

Comment:
@cdecker what am i doing wrong here? It's set L72??

https://github.com/ElementsProject/lightning/pull/4444/commits/32ae95b1c7bfbeb7bf2d8213c84705df0746b2c1#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR72

Reply:
(oh without this change it runs the default `make pytest`